api/src/apps/hooks.rs

```diff
@@ -31,6 +31,7 @@ use boa_engine::property::Attribute;
 use boa_engine::syntax::ast::node::Node;
 use boa_engine::{Context, JsValue};
 use secstr::SecUtf8;
+use serde::Deserialize;
```

Comment:
Unused? The new `use` statement in the tests mod seems to be unsed too. Please, check that

Reply:
There were a few repeated imports in tests which have been removed. The deserialize import also is removed as it is unused.